source/getting-started/javascript.md

```diff
@@ -90,48 +95,269 @@ See also: [app authorization in the API reference](/reference/#authorizing-your-
 var connection = new pryv.Connection(credentials);
 ```
 
+### Request access informations
 
-### Retrieve & manipulate events
+```javascript
+connection.accessInfo(function (err, accessInfo) {
+  //...
+});
+```
+
+### Manage events
 
 #### Retrieve
 
 ```javascript
-var filter = new pryv.Filter({limit : 20});
-connection.events.get(filter, function (err, events) {
+var filter = new pryv.Filter({limit : 10});
+connection.events.get(filter, function (err, eventList) {
   // ...
 });
 ```
 
 #### Create
 
 ```javascript
+// This is the minimum required data to create an event
 var eventData = {
-  streamId: 'diary',
+  streamId: 'valid-stream-id',
   type: 'note/txt',
-  content: 'I track, therefore I am.'
+  content: 'This is an example.'
+};
+connection.events.create(event, function (err, eventCreated) { 
+  // ...
+});
+```
+See also: [More about types](/event-types/)
+
+#### Update
+
+```javascript
+event.content = 'This is an update.';
+connection.events.update(event, function (err, eventUpdated) {
+  // ...
+});
+```
+
+#### Delete
+
+```javascript
+// Only the id of an event is needed for removal
+connection.events.delete(event, function (err, eventDeleted) {
+  // ...
+});
+```
+
+### Manage streams
+
+#### Retrieve
+
+```javascript
+var option = {
+    // If null, select all "root" streams
+    parentId: 'valid-stream-id',
+    // if null, retrieve active streams only
+    state: 'all'
+};
+connection.streams.get(option, function (err, streamList) {
+  // ...
+});
+```
+
+#### Create
+
+```javascript
+// If no id is set, one is generated;
+// If parentId is null, a "root" stream is created
+var streamData = {
+  name: 'A Stream',
+  id: 'a-stream-id',
+  parentId: 'valid-stream-id'
+};
+
+var stream = new pryv.Stream(connection, streamData);
```

Comment:
you can omit line 177. the object pryv.Stream shouldn't be instanciated by the user, but generated by the callback of connection.stream.create()